File: problems/bitManipulation/findTheDifference.java
package problems.bitManipulation;

public class findTheDifference {
    public static char method(String s, String t) {
        int res = 0;

        for (char c : s.toCharArray()) {
            res ^= (int) c;
        }
        for (char c : t.toCharArray()) {
            res ^= (int) c;
        }
        return (char) res;
    }

    public static void main(String[] args) {
        char res = findTheDifference.method("ilovemen", "ilovemnes");
        System.out.print(res);
    }
}

File: problems/bitManipulation/findTheDifference.py
def findTheDifference(s: str, t: str) -> str:
    res: int = 0
    for i  in range(len(s)):
        res ^= ord(s[i])
        res ^= ord(t[i])

    return chr(res ^ ord(t[-1]))


print(findTheDifference('ilovemen', 'imenslove'))